app/controllers/phases_controller.rb

```diff
@@ -54,4 +82,9 @@ def phase_params
   def set_project_lead
     @project_lead ||= ProjectLead.find(params[:project_lead_id])
   end
+
+  def set_engineer
+    @phase_engineers = @phase.users
+    @engineers = User.joins(:roles).where('roles.name =? ', 1)
```

Comment:
bad way of doing this. Plus this should be a scope

Reply:
Made a scope